talk/owt/sdk/conference/conferencesocketsignalingchannel.cc

```diff
@@ -418,7 +418,7 @@ void ConferenceSocketSignalingChannel::OnNotificationFromServer(
       RTC_NOTREACHED();
     }
   } else if (name == kEventNameOnSignalingMessage) {
-    RTC_LOG(LS_VERBOSE) << "Received signaling message from erizo.";
+    RTC_LOG(LS_VERBOSE) << "Received signaling message from MCU.";
```

Comment:
Please use conference server or server instead of MCU. It could work in SFU mode.

Reply:
done.